### PR TITLE
Formatting and adding final review section

### PR DESCRIPTION
**Pull request by christinaforney**
Review with "ignore whitespace" on, so you can more clearly see the change.

### CODE REVIEW

File: handbook/product/design/design_process.md [hunk 1]
```diff
@@ -16,120 +16,138 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+    - Google analytics
+
+### Refinement
+
+- Designers will review metrics and determine if their designs are meeting objectives set for them.
+- If areas for improvement are found, tickets will be created to document the change required to improve the issue
  
+### Final review of implementation
 
+- Test the feature for both themes, Light and Dark. Check for icon consistency across themes.
```
Comment (AlicjaSuska) [inline handbook/product/design/design_process.md:148]: What icon do you have in mind here? @christinaforney

Comment (rrhyne) [reply handbook/product/design/design_process.md]: I think I cleared that section up a bit.

Comment (christinaforney) [reply handbook/product/design/design_process.md]: Thanks @rrhyne!

File: handbook/product/design/design_process.md [hunk 2]
```diff
@@ -16,120 +16,139 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+    - Figma
+
+### Metrics collection
+
+- In the RFC process, the definition of success will include items which can be measured and evaluated
+- Involve the business operations team in determining what metrics to track and how they are tracked
+- Work with engineers to add the required eventLog tracking
+- Tools
+    - Looker
+    - Google analytics
```
Comment (poojaj-tech) [inline handbook/product/design/design_process.md:139]: Metrics impact many stages of product development. This should probably live under `product/analytics` once fleshed out a bit more.

**Pull request merged**